src/pages/Home.vue

```diff
@@ -151,8 +151,16 @@ export default {
     /** If sessionId exists in route, then retrieve session details. Otherwise, fallback to using group data. */
     if (this.sessionId != null) {
       this.sessionData = await sessionAPIService.getSessionData(this.sessionId);
+      // Session ID does not exist
+      if (Object.keys(this.sessionData).length == 0) {
+        this.$router.push({
+          name: "Error",
+          params: { text: "Please check Session ID" },
```

Comment:
Change to: "There is no session scheduled with this ID. Please contact your Program Manager"